tags_test.go

```diff
@@ -366,6 +366,7 @@ func TestTags_TagResource(t *testing.T) {
 		Resources: []Resource{
 			{ID: "1", Type: DropletResourceType},
 			{ID: "1", Type: ImageResourceType},
+			{ID: "1", Type: LoadBalancerResourceType},
```

Comment:
The addition doesn't affect the outcome of the test since LBs are not leveraged. I felt it could make sense to add it anyway though in case the test structure takes advantage of all supported types in the future.

Happy to remove it though if you feel it adds no (future) value.

Reply:
Hmmm... thinking out loud a bit. In general I'm a fan of the "don't add if it isn't used" rule (unused code leaves later devs afraid to remove the code because they don't know why it was added). I understand why you were tempted to add the type in this case, as you wouldn't want it to be missed by a test structure in the future.

Ok, here's my take. Either add the logic to construct `Resources` for all types or remove the explicit reference to `LoadBalancerResourceType`, whichever you prefer.

Reply:
@hilary I can totally see your point, was on the fence myself.

I'm going to remove the reference then, my thinking being that adding more types around load-balancer which aren't populated by the API yet would only add to the confusion and anxiety to modify things.

Reply:
Done -- please take a look again.